design/ceph/object-store-zone.md

```diff
@@ -0,0 +1,183 @@
+# Rook Object Store Zone
+
+## Prerequisites
+
+A rook ceph cluster with an object-store-realm and object-store-zonegroup resource created. 
+
+## Ceph Multisite Overview
+
+Multisite is a feature of Ceph that allows object stores to replicate its data over multiple Ceph clusters. 
+
+Multisite also allows object stores to be independent and isloated from other object stores in a cluster.
+
+For more information on Ceph multsite, please read the [object-store-realm](/design/ceph/object-store-realm.md) design document or [Ceph's documentation](http://docs.ceph.com/docs/master/radosgw/multisite/) on the feature.
+
+## Object Store Zone Walkthrough
+
+The resource descibed in this design document represents the zone in the Ceph Multisite data model.
+
+### Creating an Object Store Zone
+
+#### Config
+
+When the storage admin is ready to create a multisite zone for object storage, the admin will name the zone in the metadata section on the configuration file.
+
+In the config, the admin must configure the zonegroup the zone is in, whether the zone is the master zone for the zonegroup, and pools for the zone.
+
+The first object-store-zone resource created in a zonegroup must be a master zone. This means `isMaster` must be set to `True`.
```

Comment:
ideally, we don't impose order. should be possible to create the non-master first (from CRD perspective). the actual creation could wait for the master to exist